public/app/core/components/Login/ChangePassword.tsx

```diff
@@ -97,7 +97,7 @@ export class ChangePassword extends PureComponent<Props, State> {
               placeholder="New password"
               onChange={this.onNewPasswordChange}
               ref={input => {
-                this.userInput = input;
+                this.userInput = input as HTMLInputElement;
```

Comment:
There is really a lot of these null fixes. Why so many? I would assume the new code should not add that much new errors. It makes the PR harder to review.